ethcore/evm/src/instructions.rs

```diff
@@ -321,6 +321,13 @@ enum_with_from_u8! {
 		#[doc = "Makes a log entry, 4 topics."]
 		LOG4 = 0xa4,
 
+		#[doc = "Jump to a subrountine."]
+		JUMPSUB = 0xb3,
+		#[doc = "Begin a subrountine."]
+		BEGINSUB = 0xb5,
+		#[doc = "Return from a subrountine."]
+		RETURNSUB = 0xb7,
```

Comment:
https://eips.ethereum.org/EIPS/eip-2315
> We suggest the following opcodes:
0xb2 BEGINSUB
0xb3 JUMPSUB
0xb7 RETURNSUB

Reply:
Hmm, cc @gcolvin. What are the integer values of those opcodes? The one used in Geth PR and in the spec are different.

Reply:
@sorpaas I'm on the road, after midnight, on the road until midnight tomorrow, then I can check. @holiman might be able to shed some light.

Reply:
@sorpaas I think we should follow the spec https://github.com/ethereum/go-ethereum/pull/20619#issuecomment-611366159